nestkernel/archiving_node.cpp

```diff
@@ -99,7 +103,7 @@ nest::Archiving_Node::get_K_value( double t )
     if ( t > history_[ i ].t_ )
     {
       return ( history_[ i ].Kminus_
-        * std::exp( ( history_[ i ].t_ - t ) / tau_minus_ ) );
+        * std::exp( ( history_[ i ].t_ - t ) * tau_minus_inv_ ) );
```

Comment:
We'd have to run the numbers -- but we're combining root(tau, e^t1/e^t2), so it's worth looking at the relative scales of all three numbers (how big can t1, t2 and t3 get?) to see whether we can blow up in different ways. This formulation looks reasonable -- but I don't know the cases, so i can't say of the top of my head -- just asking whether it's been thought about.